.github/workflows/check_unit_tests.yml

```
@@ -74,6 +74,7 @@ jobs:
         ASAN_OPTIONS: "detect_leaks=0"
 
     - name: Code coverage
+      if: github.repository == 'musescore/MuseScore'
       if: github.event_name == 'schedule' || inputs.code_coverage == 'true'
```

Are you sure this is correct syntax, and if it is, that it is evaluated as `AND` and not as `OR`? Shouldn't it be like this:
```suggestion
      if: (github.event_name == 'schedule' && github.repository == 'musescore/MuseScore') || inputs.code_coverage == 'true'
```
(yes, that's also slightly different logic, because it does allow running coverage on forks if explicitly requested; that would make sense to me)

---

Even then it will still fail because of S3. We could split up the last step so only the push is then excluded.